google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -387,6 +412,198 @@ def snapshots token: nil, max: nil
         alias find_snapshots snapshots
         alias list_snapshots snapshots
 
+        ##
+        # Retrieves schema by name.
+        #
+        # @param [String] schema_name Name of a schema. The value can
+        #   be a simple schema ID, in which case the current project ID
+        #   will be supplied, or a fully-qualified schema name in the form
+        #   `projects/{project_id}/schemas/{schema_id}`.
+        # @param view [Symbol, String, nil] Possible values:
+        #   * `BASIC` - Include the `name` and `type` of the schema, but not the `definition`.
+        #   * `FULL` - Include all Schema object fields.
+        #
+        #   The default value is `BASIC`.
+        # @param [String] project If the schema belongs to a project other
+        #   than the one currently connected to, the alternate project ID can be
+        #   specified here. Not used if a fully-qualified schema name is
+        #   provided for `schema_name`.
+        # @param [Boolean] skip_lookup Optionally create a {Subscription} object
```

Comment:
Subscription -> Schema throughout this file.

Reply:
Fixed, thanks!